assets/img/partners/banners/partners-login.svg

```diff
@@ -1 +1 @@
eOK+KrdeqHJbhL9ZeWoANL8SfPz1K2nI9LSf0RPm5jdOFLlVBxTMr/xFFoAOIXyVL4QkmuWfHbhtC7cSoWgSkizhLKIreuR66smBJLOnaArX729AdtSXxzW1HmPZDEQyLZ8jAxm99SKbvjrNx1AuM1iPXsr17XM83qBF03LeSP/swG0NbjVC0SQkWcKpoa1DPbaiaWUZMtI5lqn1kGWzgL7lFW2sQDYlBZ3Oz6gbIbfd+E3IutUit6Iz//Qrf5GGbLDJGXapcR+i2bMRKBjR/M8/O3Db/+QWJBRNQhItnGImAaJpXWYJQPnSifvK3IqBSl4OztHNFQCF+ZCnTDz++hUdziMnmOUVedZkOsplSsmlDksWw/6YjINois5jPP+zA7dx+xKKJiHJF86TGoC84WmO4uGKZvvyhCkc5Usn7qV0BiN5KXQ3JVUBVIctdUp+MwDS6qk19Tkq3FJ95TILm2GjetdMByya3qOZJQD5jyiYhKJJyOhxnRJOiEE6gAPpOPcqmtZf16XzCUonGSe5XE4Boqdc9j8eY1GfWQIEBZNQNAkZC+HUlXC6GnIm0I5zb6LZvRhTOitKPFnTSUZNLtNKLDUpl8LD8RML0VQRzNspmISiScgYCmefur1I6jPdiGYnyxbp5AWNJFUuM1hPi097OY5iIppNyNEkCh8duJ03gYSiSQiF82QatuNwRimawutiahbprHDrktiK5YPLKQAaRCtyOeXlhi9M0RywEYiCSSiahBBH4eyY2jKRotl24TMgKlBp9stP3FPnViZR8s8eXDYbsbKt5ry++7ePtHkYotn9VAMyPV7kFiYUTUKIW+HMAiIPh+nqhtgIFNSFsoH12s7K5SfuYdSFhCyW5TQgNFMuYRmw3v0sXbGtz1yBjF6yOY9QNAkhXqXzVBYywjnT96IWhmj6u072Tv1JahBSOgFULj9O8SRBiCU0y8+U3U482Ji2sRPNkhJMTiFLKJqEkMCE02wcygKYjFEjkB/R7Fy+Wd9ZAVC9/DhT7aSvWJpjgVrEsr8oJlA0G5CzOhU+eoz1l4SiSQgJTzhTAHRDSudUokWz/wW+ATmYuRLPbRXuAWMtlakOqczA1dztfkQz8o7zFQDFjx5j/SWhaHItEDJkPi7T6jrMQaTj1QgUhGjafcYVJZ9VANXVx7cxfTii/PyXX9AAZAwhTLmc8niJ6r2vBxTNDFg0mwDKBkT+o8c4/iUhFE1CohfONAAdAvr6xTh60RwwbT6YILSzAqDeIaBMLyZHKNOQ02BqWJ8SszWOpeErKiic9/V4dZzXIIcnKn/42Be4/xJC0SQkhtI5dyoLCB19pspzvlAmTjTtLvANJZ8V9W999fjWCveQSIXSTH2bMmn+7pj+Dlo0Y1afac6yVfjwsS8wOk8IRZOQpAjn6RRkWr27ltOtaA63ESho0XS6sJuRzzUAFQNYWzu+lRf5QGTyeSWTImURyjRcTbcatGTGWjSXISOXRe41hFA0CUm6dLZ1rCdDNIX7C/rgF/ley2uuC6gwRbQKAGvH765wTwKu3mWKJACZ6gaE+heYMQYUtHiLZuD1mQ2sp8br3JsIoWgSMorS2d5A1HWhjHUj0ABnJT+1dY7vbUDAlASrfFYs37vaPHZXYmrsrt71XBoy6ggAZkTS8jsApxS3cLhZoWg2IYclKjI1TghFk5BxEk41+xCyADYnuBFo2KLZ9882332l43ErUuqw7Ir1gcN3t8qg3cJSEF1/TxmWZhu/kpVY0Qyv41zWXQpR/vCxL3DWHkIomoRQOg0Z5dQBMZ140RQ+I1HBi6ZHLxKDC7bLGwYDg6//fi+Ln2gOveO8BKD84cHfolwSQtEkhNhx7dyZNMxIp/DWwDHOomn4FayREE0R2P4Y845zNd4lygAqHx78LQ5JRAhFkxDiWjofONOWXqdoItxoZtvyxeDfe8REM6Yd5+ZwROWfMnJJCEWTEBKodGoW8ZwMTzQj6TgfTdEU8LAux6A+c7B9sGGRywrPBoRQNAkh4YunKZ0abBpLRrTjnKI5KqLZfx9cschlnUc8IRRNQkh00pnGerRTMyAm/XnMmIpmj7nqgxbNwNLmsRbNgRuBGpBDVJUBVH56cDPrLQmhaBJC4sg1DzypKfHUYDcjDDvOXYsmO85DE82mEsuKEkuOcUkIRZMQkkDpTFmkUwMwzY7zURbNWHecr7TE8tDmCo9OQiiahJBRFE/RIZ4uJIeNQOMlmgHVZ7bE8gOKJSEUTULImMrnDmuqXXRNdchGII8LGq9GoAbkbEtKLLNMhRNC0SSEEDvxfCoDOa1iBoDWf7rEERBNNgINKppmtLIKoPrBoWydRw4hFE1CCPHEhh1PaRb5zKCXfI6QaLIRCIAcIL1q/WG0khCKJiGEDEM+TelMA9AgutPuFM1hiqZvyTTT31VDCEYqCSEUTUJIzORz/ukU1qOeacu/U2GKJjvOBxLNGoA1yNR3HUD9/cNbKtx7CSEUTUJIkiVUA5Ay1uUz3S6h7DgPUDRNmaxapHLt/cNbmPYmhFA0CSHjRWr+mXSHfJo/ADDDAQIgzwAAASRJREFUjvM2mhCoquVV14VSrAGovn94C2fVIYRQNAkhZCAZXXgmAyClHmqWP2kW0XRdJ2rvd5F1nDcAUVe/m5HIzt+r7x25gxJJCKFoEkJIHJhcOGuVU6uMphxE00zvu6XStaB20awDqHecpevvHbmzzi1ECKFoEkIIIYSQseXnuAoIIYQQQghFkxBCCCGEUDQJIYQQQghFkxBCCCGEEIomIYQQQgihaBJCCCGEEIomIYQQQgghFE1CCCGEEELRJIQQQgghFE1CCCGEEEIomoQQQgghhKJJCCGEEEIomoQQQgghhFA0CSGEEEIIRZMQQgghhFA0CSGEEEIIoWgSQgghhBCKJiGEEEIIoWgSQgghhBBC0SSEEEIIIRRNQgghhBAyTvx/sRBRSJnt+dQAAAAASUVORK5CYII="/></defs></svg>
```

Comment:
In https://github.com/18F/identity-site/pull/826#discussion_r825899016, `0 0 25 25` was an example for the image with height 25x25. It may be different for others, like this with height 286x327.

Same note applies for the other `.svg`s.

Reply:
Ah I misunderstood. The svgs have been updated with the proper width and height values.